tests/Microsoft.Identity.Test.Integration/Infrastructure/SeleniumExtensions.cs

```diff
@@ -105,6 +106,19 @@ public static IWebElement WaitForElementToBeVisibleAndEnabled(this IWebDriver dr
             return continueBtn;
         }
 
+        /// <summary>
+        /// Creates a filter for selecting elements from multiple IDs. Uses XPath, e.g.
+        /// .//*[@id='otc' or @id='code']
+        /// </summary>
+        public static By ByIds(params string[] ids)
```

Comment:
this looks like we can string together any number of ids to filter by right?

Reply:
Yes, should work. Only tested with 2 though :)